web/src/containers/PatientDetails/PatientDocumentDetails/index.tsx

```diff
@@ -148,38 +152,60 @@ function PatientDocumentDetailsReadonly(props: {
                     <div className={s.buttons}>
                         {qrCompleted ? (
                             <>
-                                <Button
-                                    type="link"
-                                    onClick={() => navigate(`${location.pathname}/history`)}
-                                    className={s.button}
+                                <ConfirmActionButton
+                                    action={() => amendDocument(reload, qrId)}
+                                    reload={reload}
```

Comment:
There are two `reload` functions one on the line 156 and one one the line 157. I think one of them is redundant